src/dev/DevFlagsBox.tsx

```diff
@@ -40,11 +46,31 @@ const Gate = (flagName: string, featureFlagSettings: Record<string, string>) =>
   return null
 }
 
+const Config = (name: DynamicConfigName, savedSettings: Record<string, any>) => {
+  const statsigConfig = useDynamicConfig(name)
+  if (statsigConfig) {
+    const statsigValue = statsigConfig.getValue()
+    const setting = savedSettings[name]
+    if (setting && statsigValue !== setting) {
+      return (
+        <ThemedText.LabelSmall key={name}>
+          {name}: {JSON.stringify(setting[name])}
+        </ThemedText.LabelSmall>
+      )
+    }
+  }
+  return null
+}
+
 export default function DevFlagsBox() {
   const featureFlagsAtom = useAtomValue(featureFlagSettingsAtom)
   const featureFlags = useMemo(() => Object.values(FeatureFlag), [])
+  const dynamicConfigsAtom = useAtomValue(dynamicConfigSettingsAtom)
+  const dynamicConfigs = useMemo(() => Object.values(DynamicConfigName), [])
 
   const overrides = featureFlags.map((flagName) => Gate(flagName, featureFlagsAtom))
+  dynamicConfigs.forEach((configName) => overrides.push(Config(configName, dynamicConfigsAtom)))
```

Comment:
think you should memoize these components so they're not new elements on every render

```
const overrides = useMemo(() => {
  const featureFlagOverrides = featureFlags.map((flagName) => Gate(flagName, featureFlagsAtom))
  const dynamicConfigOverrides = dynamicConfigs.map((configName) => Config(configName, dynamicConfigsAtom))

return [...dynamicConfigOverrides, ...featureFlagOverrides].filter(Boolean) // filters out `null` items
}, [dynamicConfigsAtom, dynamicConfigs, featureFlags])
```

Reply:
and then your `hasOverrides` check can just be on the length of `overrides`

Reply:
thought about this but is hard to memoize bc Gate & Config call hooks